package.json

```diff
@@ -22,6 +22,7 @@
     "passport": "^0.4.1",
     "passport-github": "^1.1.0",
     "passport-oauth2": "^1.5.0",
+    "pg": "^8.0.0",
```

Comment:
Is there typescript support to pgpool? If so it might be best to add it 😃

Reply:
Also... but not less important.

Do we need pg that is version 10, like in the army?

So we won't meet in the future... Versions problem 😃

Reply:
version 8 is the newest:  
![image](https://user-images.githubusercontent.com/62829761/78144532-633aa600-7438-11ea-8847-8c41a9dadde2.png)

Reply:
What do we have in the army then 🤔 ?

@adirsiton

Reply:
I think you're talking about the postgres version it's not the same

Reply:
`7.14` (@stavbi1 Helped 😃 )

If posible, change to:
`"pg": '7.14.0' // Army compatibility`

Reply:
I assume will whiten a docker image, and we won't whiten the whole project so we don't need to think about army env compatibility.

Reply:
Ok coo!! 😃 .

Interesting point on why not the whole project (Excluding binaries of course...)